documentation/source/users/rmg/input.rst

```diff
@@ -549,6 +552,12 @@ Miscellaneous options::
         saveEdgeSpecies=True,
     )
 
+The ``name`` field is the name of any generated seed mechanisms
+
+Setting ``generateSeedEachIteration`` to ``True`` tells RMG to save and update a seed mechanism and thermo library during the current run
```

Comment:
If this is 'off', does this create a seed at the end of the simulation? What is the intended behavior?